### PR TITLE
More escaping examples

### PR DESCRIPTION
**Pull request by hadley**

### CODE REVIEW

**Comment by hadley** [comment]
Fixes #990

File: src/escapeExamples.cpp [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      } else if (*cur == '\'' || *cur == '"' || *cur == '`') {
+        in_string = *cur;
+      }
+    } else { // inside string/symbol
+      if (in_escape) {
+        in_escape = false;
+        if (*cur == 'l' || *cur == 'v') {
+          out += '\\';
+        } else if (*cur == '\\') {
+          out += "\\\\";
```
Comment (gaborcsardi) [inline src/escapeExamples.cpp]: So this will add two backslashes, with `*cur` it is three, which does not seem right. Why does this work?

Comment (gaborcsardi) [reply src/escapeExamples.cpp]: Never mind, got it finally. We wrote out the original escape already, and three more is four.

Comment (hadley) [reply src/escapeExamples.cpp]: Yeah, I don't understand it either but empirically it works.

**Pull request merged**